stackslib/src/chainstate/stacks/db/transactions.rs

```diff
@@ -8687,14 +8687,14 @@ pub mod test {
                         epoch_id: StacksEpochId::Epoch2_05,
                         start_height: 1,
                         end_height: 2,
-                        block_limit: HELIUM_BLOCK_LIMIT_20.clone(),
+                        block_limit: LOCAL_TESTNET_BLOCK_LIMIT_20.clone(),
```

Comment:
I would suggest a more explicit naming as it adds extra terms, while its goal is of removing them. 
eg. REGTEST_BLOCK_LIMIT or REGTEST_OR_MOCKNET_BLOCK_LIMIT. 

I am open to any suggestions.

Reply:
@ASuciuX regtest and mocknet refer to local testnet nodes, hence why I suggested reconciling to "local" whenever "helium" was being used interchangeably for both. Past the CLI and into the runloops, regtest and mocknet are used for the respective Bitcoin burnchains. Reconciling might help create that boundary between the Stacks node and its burnchains. With that said, I'm okay with renaming to `REGTEST_OR_MOCKNET_BLOCK_LIMIT`. I don't think there will be other chains that would complicate this convention (e.g. `REGTEST_OR_MOCKNET_OR_SIMNET_BLOCK_LIMIT`). Thoughts?